aspnet/migration/rc1-to-rc2.rst

```diff
@@ -265,41 +332,37 @@ We now consider a type to be a controller if all of the following rules apply:
 - It's important to note that if ``[NonController]`` is applied anywhere in the type hierarchy the discovery conventions will never consider that type or its descendants to be a controller. ``[NonController]`` takes precedence over ``[Controller]``.
 
 
-DNX commands replaced with .NET Core tools
-------------------------------------------
-
-web, user-secrets, sql-cache, watch.
-
-
 Configuration
 -------------
 
-``IConfigurationSource`` has been introduced to represent the settings/configuration which is used to ``Build`` an IConfigurationProvider. It is no longer possible to access the provider instances from IConfigurationBuilder only the sources. This is intentional, but may cause loss of functionality as you can longer do things like explicitly call ``Load`` on the provider instances.
+``IConfigurationSource`` has been introduced to represent the settings/configuration which is used to ``Build()`` an ``IConfigurationProvider``. It is no longer possible to access the provider instances from ``IConfigurationBuilder`` only the sources. This is intentional, but may cause loss of functionality as you can longer do things like explicitly call ``Load`` on the provider instances.
 
-``FileConfigurationProvider`` base class has been introduced as a common root for Json/Xml/Ini providers. This allows the ability to specify an ``IFileProvider`` on the source which will be used to read the file instead of explicitly using ``File.Open``. The side effect of this change is that absolute paths are no longer supported. The file path must be relative to the base path of the ``IConfigurationBuilder``'s basepath or the ``IFileProvider``, if specified.
+``FileConfigurationProvider`` base class has been introduced as a common root for Json/Xml/Ini providers. This allows the ability to specify an ``IFileProvider`` on the source which will be used to read the file instead of explicitly using ``File.Open()``. The side effect of this change is that absolute paths are no longer supported. The file path must be relative to the base path of the ``IConfigurationBuilder``'s basepath or the ``IFileProvider``, if specified.
 
 	   
-Entity Framework Core
----------------------
+Entity Framework
```

Comment:
This section seems buried in a sea of ASP.NET Core changes. Can we remove this section and instead add the link in the Overview section at the top?